d2l-localize-mixin.html

```diff
@@ -0,0 +1,18 @@
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="d2l-localize-behavior.html">
+
+<script>
+	window.D2L = window.D2L || {};
+	window.D2L.PolymerMixins = window.D2L.PolymerMixins || {};
+
+	/**
+	* @polymerMixin
+	* @appliesMixin D2L.PolymerBehaviors.LocalizeBehavior
+	*/
+	D2L.PolymerMixins.LocalizeMixin = Polymer.dedupingMixin(
```

Comment:
Here, we first use the `Polymer.mixinBehaviors` method to expose legacy behavior as mixin.  Then we wrap the result in the `Polymer.dedupingMixin`.

The Polymer docs recommend using the `dedupingMixin` on shared mixins to prevent them from being duplicated in the class hierarchy.  Rather than try to reiterate, the details are [here](https://www.polymer-project.org/2.0/docs/devguide/custom-elements#packaging-mixins-for-sharing).